projects/fluentci.io/package.yml

```diff
@@ -0,0 +1,29 @@
+distributable:
+  url: https://github.com/fluentci-io/fluentci/archive/refs/tags/v{{ version }}.tar.gz
+  strip-components: 1
+
+display-name: fluentci
+
+versions:
+  github: fluentci-io/fluentci
+
+dependencies:
+  dagger.io: ^0.8.8
+  charm.sh/glow: ^1.5.1
+
+build:
+  dependencies:
+    deno.land: ^1.37
+  working-directory: ${{prefix}}/bin
+  script: |
+    deno compile -A --unstable --import-map="$SRCROOT"/import_map.json --output fluentci "$SRCROOT"/main.ts
+    mkdir -p "{{ prefix }}"/bin
+    cp fluentci "{{ prefix }}"/bin
+  # ^^^^ yes, this is a hack, but it's the only way to get the test to run, and it works
+  # the test fail if we don't copy the binary to the prefix, I don't know why
```

Comment:
what about dropping the working-directory and just doing:
```yaml
  script:
    - deno compile -A --unstable --import-map=import_map.json --output fluentci main.ts
    - mkdir -p "{{ prefix }}"/bin
    - install fluentci "{{ prefix }}"/bin
```

Reply:
I think I've found the problem, we also need to skip this package [here ](https://github.com/pkgxdev/brewkit/blob/a9c140eec89c8a9db429c5c2ffb2c2b03198bbf3/share/brewkit/fix-elf.ts#L26)

Reply:
I created a PR for this : https://github.com/pkgxdev/brewkit/pull/189